src/components/form_input/FormInput.jsx

```diff
@@ -0,0 +1,23 @@
+import "./FormInput.css";
+const FormInput = ({ label, value, onChangeFunction, type, required }) => {
+	return (
+		<div>
+			<div className="input-container">
+				<label className="label text-bold" htmlFor={label}>
```

Comment:
If you are trying to dynamically give className, then I think it should be like this -
className=\` ${label} text-bold\`.
And I am unable to find class label in any CSS files

Reply:
htmlfor is for connecting the label with the input.